ticketmaster.js

```diff
@@ -59,15 +59,48 @@ async function getEventsByArtist(artist) {
   
   // use attraction ids to search to specific events
   response = await fetch(`${eventSearchUrl}&attractionId=${attractionIDs}`)
-  result = await response.json()
-  const events = result._embedded.events
+  result = await response.json();
+  //searching for certain artist throws a TypeError , whereas some will return successfully
```

Comment:
nice! good call. Figured we should add error handling for the api calls but was gonna do that in a future PR
Out of curiosity, which searches returned a typeerror? Want to test this out

Reply:
We think this happens when you search for an artist that does not have any shows. For example, "Kanye West" yields an error because he is not touring.